src/common/types/vector.cpp

```diff
@@ -838,7 +838,7 @@ void Vector::UTFVerify(idx_t count) {
 	UTFVerify(FlatVector::INCREMENTAL_SELECTION_VECTOR, count);
 }
 
-void Vector::Verify(const SelectionVector &sel, idx_t count) {
+void Vector::Verify(const SelectionVector &sel, idx_t count, ValidityMask *validity_parent) {
```

Comment:
Can you revert this change? This will only work on nested structures with depth=1. I am fixing this differently in the struct storage branch.